partner/app/tests/test_app.py

```diff
@@ -212,3 +212,34 @@ def test_read_tilesp_validate_country_code(client, country_code):
     assert "status" in response_content
     assert "count" in response_content
     assert "response" in response_content
+
+
+def test_read_tilesp_validate_unknown_query_params(client):
```

Comment:
I don't think `validate` is correct in this case since we don't check whether the value is of the correct format etc. but return an error if we don't recognize the query parameter:

```suggestion
def test_read_tilesp_error_for_unknown_query_params(client):
```